src/main/java/mate/academy/bookstore/model/Book.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.bookstore.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private String author;
+    private String isbn;
+    private BigDecimal price;
+    private String description;
+    private String coverImage;
```

Comment:
Please, pay attention that your entity fields should have more constraints:
<img width="276" alt="image" src="https://github.com/BohdanTrue/book-store-spring/assets/140437083/a054ff5c-c091-4156-92b4-e299071cf5e7">